src/Base/RestAPI/ItemFields/SeoPress.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Adds portal url field to the output.
+ */
+
+namespace OWC\OpenPub\Base\RestAPI\ItemFields;
+
+use OWC\OpenPub\Base\Models\Item;
+use OWC\OpenPub\Base\Support\CreatesFields;
+use WP_Post;
+
+class SeoPress extends CreatesFields
+{
+    /**
+     * The condition for the creator.
+     *
+     * @return callable
+     */
+    protected function condition(): callable
+    {
+        return function () {
+            return true;
```

Comment:
Hier zou ik checken of de SeoPress plugin geïnstalleerd is. 
`return \is_plugin_active('plugin-folder/main-plugin-bestand.php')`